src/browser/index.jsx

```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
 });
 
 // ErrorCode: https://code.google.com/p/chromium/codesearch#chromium/src/net/base/net_error_list.h
-// FIXME: need better wording in English
+const errorPage = {
+  tableStyle: {
+    display: 'table',
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    top: '-70px',
+    left: '0'
+  },
+
+  cellStyle: {
+    display: 'table-cell',
+    verticalAlign: 'middle'
+  },
+
+  bullets: {
+    paddingLeft: '15px',
+    lineHeight: '1.7'
+  },
+
+  techInfo: {
+    fontSize: '12px',
+    color: '#aaa'
+  },
+};
+
 var ErrorView = React.createClass({
   render: function() {
     return (
       <Grid id={ this.props.id } style={ this.props.style }>
-        <h1>Failed to load the URL</h1>
-        <p>
-          { 'URL: ' }
-          { this.props.errorInfo.validatedURL }
-        </p>
-        <p>
-          { 'Error code: ' }
-          { this.props.errorInfo.errorCode }
-        </p>
-        <p>
-          { this.props.errorInfo.errorDescription }
-        </p>
-        <p>Please check below. Then, reload this window. (Ctrl+R or Command+R)</p>
-        <ListGroup>
-          <ListGroupItem>Is your computer online?</ListGroupItem>
-          <ListGroupItem>Is the server alive?</ListGroupItem>
-          <ListGroupItem>Is the URL correct?</ListGroupItem>
-        </ListGroup>
+        <div style={ errorPage.tableStyle }>
```

Comment:
By removing these two `<div>`, tabs properly works while showing the error page. So you might need to adjust the vertical position with other ways.